src/geojson/geojson_intf.ml

```diff
@@ -216,12 +199,30 @@ module type S = sig
     end
   end
 
-  type t =
+  type geojson =
     | Feature of Feature.t
     | FeatureCollection of Feature.Collection.t
     | Geometry of Geometry.t  (** A geojson object *)
 
-  include Json_conv with type t := t and type json := json
+  (* type t = { geojson : geojson; bbox : float array option } *)
+  type t
+
+  val geojson : t -> geojson
+  (** [geojson t] will extract geojson value from t (a GeoJSON object) *)
+
+  val bbox : t -> float array option
+  (** [bbox t] will extract bbox value from t (a GeoJSON object) *)
+
+  val typ_t : geojson -> float array option -> t
```

Comment:
I think this would be better using optional labels and to be consistent with the rest of the library, we can use the name `v`:

```suggestion
  val v : ?bbox:float array -> geojson -> t
```